README.md

```diff
@@ -36,6 +36,7 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Andrew Kelley](#andrew-kelley) - **streaming:** Zig programming language, Compiler, LLVM, ASM
 - [Anthony Sottile](#anthony-sottile) - **streaming:** Python, AST manipulations
 - [Antonio Maiorano](#antonio-maiorano) - **streaming:** Emulator Development, C++
+- [Ardalis - Steve Smith](#ardalis) - **streaming:** C#, ASP.NET Core, Clean Architecture, DDD, Design Patterns, TDD
```

Comment:
This anchor link `#ardalis` is not going to work. It needs to match the generated anchor link produced by the GitHub rendering of README in order to link internally on the page properly. The correct anchor is `#ardalis---steve-smith`.

Reply:
Should be fixed.